src/ui/occupantswin.c

```diff
@@ -46,18 +46,37 @@ _occuptantswin_occupant(ProfLayoutSplit *layout, Occupant *occupant, gboolean sh
     theme_item_t presence_colour = theme_main_presence_attrs(presence_str);
     wattron(layout->subwin, theme_attrs(presence_colour));
 
-    GString *msg = g_string_new("   ");
+    GString *spaces = g_string_new(" ");
+
+    int indent = prefs_get_occupants_indent();
+    int current_indent = 0;
```

Comment:
What is the purpose of `current_indent`?

Reply:
Right, going to remove that. I thought about passing this to win_sub_print() at first.